### ISSUE TITLE
Any Copy/Cut operation doesn't work throughout Visual Editor

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#12506]
Regression in ~~4.6.0~~ master.

To test, make sure you have latest master (at the time of writing it's 4.6.1 https://github.com/WordPress/gutenberg/commit/68367b4b9c869900eca07b9a9b3750ee52915c46) ~~or at least 4.6.0~~.

- go in any post
- select some text in a block editable area, either with mouse or keyboard doesn't matter, and try to paste it in another block or in an external editor
- nothing happens (if you have previous content in your clipboard, that will be pasted)
- try to cut text: nothing happens
- try also in the post title: same as above
- try also with an embed, for example a vimeo embed: paste `https://vimeo.com/22439234` then edit it e.g. remove a few numbers, copy, paste it: the previous full URL get pasted
- try to cut the vimeo URL: nothing happens
- try even to select some text in the editor UI, outside of any block, for example the paragraph block description:

<img width="301" alt="screenshot 2018-12-01 at 17 49 27" src="https://user-images.githubusercontent.com/1682452/49331740-6868b600-f5a2-11e8-9aa7-5460fb62ee7b.png">

- copy and paste it: same as above
- switch to Code Editor mode: everything works normally

Basically any basic Copy / Cut operation in the Visual Editor is currently broken.

Seems to me this commit https://github.com/WordPress/gutenberg/commit/3ce3f0380ad77ba0b9df8c5a94741822ce8bbbb5 made yesterday – Friday, Nov 30, 2018 – (see https://github.com/WordPress/gutenberg/pull/11851) with the changes to the `CopyHandler` component, is responsible for this breakage.

**Comment by afercia** [comment]
I'd like to add some considerations. While this regression is probably easy to fix, it's worth considering this kind of things shouldn't happen at this point of the release cycle.

There's some disagreement between the most active contributors about the Gutenberg development process in relationship with the WordPress release process. Personally, I'd tend to think that at this point of the release cycle Gutenberg should follow the same rules that WordPress follows in the release cycle.

WordPress is now in Release Candidate stage: that means [no commits should be made other than the ones necessary to fix bugs reported against it](https://make.wordpress.org/core/handbook/about/release-cycle/releasing-major-versions/#release-candidate) (or small changes related to documentation). When bugs reported against a RC get fixed, a new RC should be released.

Gutenberg is the _only_, big, relevant feature of WordPress 5.0. At this point of the release cycle, it should be considered part of WordPress and subject to same rules. Instead, in the last days a relevant amount of commits has been merged in Gutenberg.

Honestly, with all the respect due to the hard work and dedication of everyone involved in this project, I don't see how WordPress 5.0 can be considered in a "Release Candidate" stage while dozens of commits get merged in a big piece of software that is part of it. Seems to me this process is not wise, unreasonable, and prone to introduce other regressions. WordPress has a responsibility towards 32% of the Web and millions of users. I'd kindly invite everyone to reconsider the current process, stop introducing changes during Release Candidate, and consider a decent amount of time to test _deeply_ this software before release.

**Comment by gziolo** [comment]
> WordPress is now in Release Candidate stage: that means no commits should be made other than the ones necessary to fix bugs reported against it (or small changes related to documentation). When bugs reported against a RC get fixed, a new RC should be released.

There is `4.6` release branch created for RC cycle. `master` is meant to contain code planned for WorPress `5.0.1` release.

**Comment by gziolo** [comment]
> However, several commits have been merged in 4.6.0 since November 23rd (first RC day) so the general consideration still stands.

Once again, we are not using `master` for RC anymore. WordPress RC2 contains what we released in Gutenberg 4.5 + cherry-picked commits for 4.6 and 4.6.1 as listed on make.wordpress.org.

I think I know why it has regressed, I will fix tomorrow. Thanks for reporting it.

**Comment by gziolo** [comment]
Opened #12543 with a proposed fix.

**Comment by davisshaver** [comment]
Ran into this today. I agree with @afercia's comments (couldn't believe this was an actual regression and I thought my computer was broken) but think we might just need a new way to pin our builds to the next release. I wonder if we might want something like @danielbachhuber's [nightly builds](https://builds.danielbachhuber.com/gutenberg-nightly.zip) but for the branch linked to the next release?
